x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/data_quality_panel/data_quality_summary/summary_actions/check_all/check_index.test.ts

```diff
@@ -164,51 +163,6 @@ describe('checkIndex', () => {
     });
   });
 
-  describe('when `ecsMetadata` is null', () => {
-    const onCheckCompleted = jest.fn();
-
-    beforeEach(async () => {
-      jest.clearAllMocks();
-
-      await checkIndex({
-        abortController: new AbortController(),
-        batchId: 'batch-id',
-        checkAllStartTime: Date.now(),
-        ecsMetadata: null, // <--
-        formatBytes,
-        formatNumber,
-        httpFetch,
-        indexName,
-        isLastCheck: false,
-        onCheckCompleted,
-        pattern,
-        version: EcsVersion,
-      });
-    });
-
-    test('it invokes onCheckCompleted with a null `error`', () => {
-      expect(onCheckCompleted.mock.calls[0][0].error).toBeNull();
-    });
-
-    test('it invokes onCheckCompleted with the expected `indexName`', () => {
-      expect(onCheckCompleted.mock.calls[0][0].indexName).toEqual(indexName);
-    });
-
-    test('it invokes onCheckCompleted with the default `partitionedFieldMetadata`', () => {
-      expect(onCheckCompleted.mock.calls[0][0].partitionedFieldMetadata).toEqual(
-        EMPTY_PARTITIONED_FIELD_METADATA
-      );
-    });
-
-    test('it invokes onCheckCompleted with the expected `pattern`', () => {
-      expect(onCheckCompleted.mock.calls[0][0].pattern).toEqual(pattern);
-    });
-
-    test('it invokes onCheckCompleted with the expected `version`', () => {
-      expect(onCheckCompleted.mock.calls[0][0].version).toEqual(EcsVersion);
-    });
-  });
-
```

Comment:
For reviewers:

This test was removed because it tests an impossible state. Details:
https://github.com/elastic/kibana/pull/185025/files#r1631554934